### PR TITLE
fix: Prevent grid blowouts in the Wizard component

### PR DESCRIPTION
**Pull request by DaemonCahill**
### Description

A customer issue was received that used a large `Table` component in the `content` slot of one of the `Wizard` component `steps`. This caused a grid blowout that pushed the content beyond the containing grid definition in the `Wizard` component. The resolution was to wrap the second column definition in a `minmax()` function.

**Before:**
blowout

**After:**
resolved

### How has this been tested?

The fix was manually tested in all major browsers. In addition, a new test page was added to the `AppLayout` category called `with-wizard-and-table`.

### Documentation changes

*Do the changes include any API documentation changes?*
- [ ] _Yes, this change contains documentation changes._
- [x] _No._

### Related Links

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- [x] _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](CONTRIBUTING.md#public-apis)._
- [x] _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](CONTRIBUTING.md#browsers-support)._
- [x] _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- [x] _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- [ ] _Changes are covered with new/existing unit tests?_
- [x] _Changes are covered with new/existing integration tests?_
</details>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pages/app-layout/with-wizard-and-table.page.tsx [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                      ariaLabels={{
+                        selectionGroupLabel: 'Items selection',
+                        allItemsSelectionLabel: ({ selectedItems }) =>
+                          `${selectedItems.length} ${selectedItems.length === 1 ? 'item' : 'items'} selected`,
+                        itemSelectionLabel: ({ selectedItems }, item) => {
+                          const isItemSelected = selectedItems.filter(i => i.name === item.name).length;
+                          return `${item.name} is ${isItemSelected ? '' : 'not'} selected`;
+                        },
+                      }}
+                      columnDefinitions={[
```
Comment (abdhalees) [inline pages/app-layout/with-wizard-and-table.page.tsx:58]: there is some shared configs and generated items for tables in these files
https://github.com/cloudscape-design/components/blob/main/pages/table/shared-configs.tsx
https://github.com/cloudscape-design/components/blob/main/pages/table/generate-data.ts

**Review by abdhalees — APPROVED** [review]

**Review by pan-kot — COMMENTED** [review]
I think it makes sense adding a screenshot tests for this page.

**Review by pan-kot — DISMISSED** [review]

File: pages/app-layout/with-wizard-and-table.page.tsx (hunk 1 above)
Comment (DaemonCahill) [reply pages/app-layout/with-wizard-and-table.page.tsx]: Excellent, thank you!

**Comment by DaemonCahill** [comment]
@pan-kot @abdhalees I'll add the new test page to the integration test repo after this is merged.

**Review by abdhalees — DISMISSED** [review]

**Review by pan-kot — APPROVED** [review]

**Review by abdhalees — APPROVED** [review]

**Pull request merged**